content/images/post-images/2018-09-12-2018-community-survey/dummy.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Purpose unclear.

Reply:
This was there simply to allow me to create an empty folder in the UI